muon/muesr/examples/test/example.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
We should not edit files if they are already 2/3 compatible and many other files like this